modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -3,98 +3,111 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="creating-cluster-id-labels-for-metrics_{context}"]
-= Creating cluster ID labels for metrics
 
-ifndef::openshift-dedicated,openshift-rosa[]
-You can create cluster ID labels for metrics for default platform monitoring and for user workload monitoring.
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="creating-cluster-id-labels-for-metrics-cpm_{context}"]
+= Creating cluster ID labels for metrics for core platform monitoring
+// end::CPM[]
+
+// tag::UWM[]
+[id="creating-cluster-id-labels-for-metrics-uwm_{context}"]
+= Creating cluster ID labels for metrics for monitoring of user-defined projects
+// end::UWM[]
```

Comment:
So just something to keep in mind. This seems to work on the preview, but I'm told that tags can sometimes be a little weird when built for docs.redhat.com. So it's fine as is but I'd advise taking a close look on docs.redhat whenever you do merge this content into our live branches.

Or if you know of examples where this exact tag use case already exists, it would be good to look at that